### ISSUE TITLE
TWE Design System Components: Date Pickers

### ISSUE THREAD

**Issue by RobinElayn** [hackforla/internship#419]
### Overview
We need to create components for the TWE design system for the future website development & handoff.

### Action Items
- [x] Create mood board 
- [x] Present to the team
- [x] Get feedback
- [x] Incorporate feedback
- [x] Hi-Fi version
- [x] Create variations
- [x] Create organized component
- [x] Get sign off from product lead 
- [x] When this issue is complete, please go #397 and check off the dependency.  If all other dependencies are checked off, move the issue from the icebox to the new issue approval column.
- [ ] Create/indicate microsite for the dev team

### Resources/Instructions
- [Google Doc](https://docs.google.com/document/d/14BZYFEa5s5FESeASNorEDLH6zEW0LTYcpD1NVyE66iM/edit#heading=h.pq5p3t1w1lw2)
- [HfLA Figma File](https://www.figma.com/file/UvUJ0zT2EEIOoDQVdn6ApN/HfLA-Design-System-Template?t=QhvqYQr8sPVUXqCz-0)
- [Internships Figma File](https://www.figma.com/file/TTRS2FWXsrymHYpPJL1IdH/Internship-Team-Main-file?node-id=2-45&t=UnquUaQhvcuGgsKH-0)

**Comment by meetminji** [comment]
Check [Internships Figma File](https://www.figma.com/file/TTRS2FWXsrymHYpPJL1IdH/Internship-Team-Main-file?node-id=2-45&t=UnquUaQhvcuGgsKH-0)

**Comment by RobinElayn** [comment]
Updated styles within component. Checked that all variations were properly linked to Master Component.

**Comment by meetminji** [comment]
Updated states 

<img width="784" alt="date picker" src="https://github.com/hackforla/internship/assets/98001179/81d31daa-7b2c-49af-84cc-01be6cc31640">

**Comment by ExperimentsInHonesty** [comment]
@joshfishman will add a screen shot of picking the dates.

**Comment by joshfishman** [comment]
<img width="337" alt="Screenshot 2023-10-12 at 10 46 00 AM" src="https://github.com/hackforla/internship/assets/1108221/63496b03-520e-4620-a4d1-e99f8c01c77a">
[Material UI Datepicker](https://mui.com/x/react-date-pickers/date-picker/)

**Comment by meetminji** [comment]
Created a calendar for another method to pick date

<img width="917" alt="date picker with calendar" src="https://github.com/hackforla/internship/assets/98001179/5a832eda-24cb-4bd8-8aed-7286706a4a25">

**Comment by meetminji** [comment]
Today's date "17" changed to extrabold 

<img width="598" alt="date picker with calendar 2" src="https://github.com/hackforla/internship/assets/98001179/b7340587-589c-460a-a784-0c2ac4d62c01">

**Comment by meetminji** [comment]
final 

<img width="672" alt="date picker final" src="https://github.com/hackforla/internship/assets/98001179/47b8bde3-e876-417f-8a34-8a974b9c7504">

**Comment by karina-sato** [comment]
We need to add the descriptive details and create a microsite for this component

**Comment by mklmrgn** [comment]
Re-opening this to use our new M3 component Atoms and M3 style

**Comment by jenregis91** [comment]
@ExperimentsInHonesty Yes I'm going to be leaving the team. I'm going to be starting a new job, working full-time, so unfortunately I'm going to have to stop my internship. I remembered there being mention of an off-boarding process, so I wanted to know what my next step is?

**Comment by Rabia2219** [comment]
#### 2025-09-30

- Offboarded Jennifer, unassigned her from this issue and moved it to New Issue Approval
